source/software/matlab_parallel_computing.rst

```diff
@@ -3,12 +3,126 @@
 MATLAB parallel computing
 =========================
 
-Matlab has several products to facilitate parallel computing, e.g.
-
--  The `Parallel Computing
-   Toolbox <https://nl.mathworks.com/products/parallel-computing.html>`_
-   is a regular Matlab toolbox that lets you write parallel Matlab
-   applications or use parallel implementations of algorithms in other
-   toolboxes.
-   Try ``help distcomp`` to see if the toolbox is installed for the
-   version of Matlab that you're using.
+Matlab allows you to use parallel computing on a HPC cluster using the `Parallel Computing Toolbox <https://www.mathworks.com/products/parallel-computing.html>`_.
+Only Matlab installations that have this installed, can be configured to use this. This documentation will describe how to configure this. Be aware that this has only
+been tested for some installations on the KU Leuven Tier-2 clusters.
```

Comment:
The main issue here is that the scripts I have are created for Genius specifically. So while I expect that other sites might have a similar approach, I cannot say for sure that this will actually be the case. So for now it would seem that the general part would be only the info of the parallel computing toolbox. I can of course add an extra note that states that this info is only applicable to Genius for the time being

Reply:
OK, but even in that case I think it's good to separate the two,
i.e. to keep that one parallel computing toolbox paragraph in
a first "general" section and then put all the rest in a KU Leuven
section (which then gets further divided in subsections etc.)

Reply:
I have made some adaptations, should be oknow